cove_ocds/management/commands/ocds_cli.py

```diff
@@ -30,10 +31,21 @@ def handle(self, file, *args, **options):
             ))
 
         try:
-            result = ocds_json_output(self.output_dir, file, schema_version, convert)
+            if os.path.isdir(file):
+                with open(os.path.join(self.output_dir, "all_results.txt"), 'w+') as outfile:
+                    for f in os.listdir(file):
+                        filename = os.fsdecode(f)
+                        if filename.endswith(".json"):
```

Comment:
No big deal, but the code in this repo normally uses single quotations marks (although TBH that is not 100% consistent, with myself one of the persons to blame)